platform/tezos/client.go

```diff
@@ -49,3 +49,11 @@ func (c *Client) GetBlockByNumber(num int64) ([]Tx, error) {
 
 	return list, err
 }
+
+func (c *Client) GetAccount(address string) (result Account, err error) {
+	path := fmt.Sprintf("node_account/%s", address)
+	referer := fmt.Sprint("https://tzscan.io/%s", address)
```

Comment:
why is this needed? the rest seems to work without this change

Reply:
This particular api returns 403 without this header. It was tricky part of that integration )